webhooks/pod_webhook.go

```diff
@@ -178,6 +178,10 @@ func (m *PodMutator) injectSidecar(pod *corev1.Pod, configMap string, featureFla
 		envs = featureFlag.Spec.FlagDSpec.Envs
 	}
 
+	// Add additional environment variables to existing containers
+	for _, container := range pod.Spec.Containers {
```

Comment:
`range` copies the slice value, this doesn't change the value in the slice itself.

Something like this would work:
```
for i := 0; i < len(pod.Spec.Containers); i++ {
		cntr := pod.Spec.Containers[i]
		cntr.Env = append(cntr.Env, featureFlag.Spec.FlagDSpec.Envs...)
		pod.Spec.Containers[i] = cntr
	}
```

Reply:
You are absolutely right

Reply:
I'm confused, why did this get merged?

Reply:
Looks like an error. 
I have the change on my HEAD locally here. Apologies will open and resolve
![Screenshot 2022-08-24 at 10 41 22](https://user-images.githubusercontent.com/1235925/186386381-dc9ab1af-bf2f-4982-942c-dfe11ed4840c.png)
.

Reply:
https://github.com/open-feature/open-feature-operator/pull/79